ui/webui/test/helpers/installer.py

```diff
@@ -16,51 +16,55 @@
 # along with this program; If not, see <http://www.gnu.org/licenses/>.
 
 
-class Installer():
-    welcome_id = "installation-language"
-    storage_id = "installation-destination"
-    review_id = "installation-review"
-    progress_id = "installation-progress"
-    steps = [welcome_id, storage_id, review_id, progress_id]
+class InstallerSteps():
+    welcome = "installation-language"
+    storage = "installation-destination"
+    review = "installation-review"
+    progress = "installation-progress"
```

Comment:
These are actually constants and used that way, so it would make sense to have them uppercased.